projects/uswds-components/src/lib/step-indicator/step-indicator.component.html

```diff
@@ -29,7 +29,7 @@
     </li>
   </ol>
 
-  <div *ngIf="headerPosition === 'bottom'" class="usa-step-indicator__header">
+  <div *ngIf="!headerPosition || headerPosition === 'bottom'" class="usa-step-indicator__header">
```

Comment:
Is this really needed? headerPosition is defined as either 'top' or 'bottom' and has a default value defined, so there shouldn't ever be a scenario where it is set to a falsy value.

Reply:
Updated to remove the redundant check